DEVELOPMENT.md

```diff
@@ -105,15 +105,29 @@ releases and architectures like this:
 
 ## Launch Sippy API
 
-If you are *not* loading a backup for your data, you will need to initialize and/or update the database schema. This step is done automatically when fetching data from testgrid or prow, but may not have run if you start the server before doing so.
+If you are *not* loading a backup for your data, you will need to
+initialize and/or update the database schema:
 
+```bash
+./sippy migrate
+```
+
+Then to launch the API server:
 ```bash
 ./sippy serve \
   --log-level=debug \
   --database-dsn="postgresql://postgres:password@localhost:5432/postgres" \
   --mode=ocp
 ````
 
+If you'd like to launch just Component Readiness, you can run:
```

Comment:
optional, but I believe that the `~/google-service-account-credential-file.json` must allow access to a dataset called `ci_analysis_us` within a project called `openshift-gce-devel` project because we hardcode that; it would be good to include that in this doc.

Also, you can put a json that doesn't have the access and we will show logs like:

```
INFO[2024-04-02T15:15:20.542-04:00] beginning refresh metrics                    
ERRO[2024-04-02T15:15:20.989-04:00] error querying releases data from bigquery    error="googleapi: Error 403: Access Denied: Project openshift-gce-devel: User does not have bigquery.jobs.create permission in project openshift-gce-devel., accessDenied"
ERRO[2024-04-02T15:15:20.989-04:00] error getting releases from bigquery          error="googleapi: Error 403: Access Denied: Project openshift-gce-devel: User does not have bigquery.jobs.create permission in project openshift-gce-devel., accessDenied"
ERRO[2024-04-02T15:15:20.989-04:00] error refreshing metrics                      error="googleapi: Error 403: Access Denied: Project openshift-gce-devel: User does not have bigquery.jobs.create permission in project openshift-gce-devel., accessDenied"
INFO[2024-04-02T15:15:20.99-04:00] Serving reports on :8080
```

but this will still serve on 8080 ut will get 500s when trying to goto http://localhost:3000/sippy-ng/component_readiness/main